packages/account-tree-controller/src/AccountTreeController.test.ts

```diff
@@ -2451,4 +2451,204 @@ describe('AccountTreeController', () => {
       expect(spy).toHaveBeenCalledWith(groupId, hidden);
     });
   });
+
+  describe('Event Emissions', () => {
+    it('should NOT emit accountTreeUpdated when tree is initialized', () => {
```

Comment:
Prefer the imperative form for tests:

```suggestion
    it('does NOT emit accountTreeUpdated when tree is initialized', () => {
```

NOTE: If you could apply all this to every tests as well, this would be awesome!